include/common/s1ap_structs.h

```diff
@@ -39,177 +41,16 @@
 #define NI_DTCH_REQUEST_NO_OF_IES 3
```

Comment:
This should be moved as well.

Reply:
Nope. This is number of IEs in the s1ap protocol. Hence it should remain in s1ap_Strcut file. Let me know @asreek  if you don't agree.

Reply:
Got it. We can retain it here.